tests/test_faster_rcnn.py

```diff
@@ -99,15 +104,20 @@ def test_faster_rcnn_train_all(mock_create, mock_loss, config, dataset):
     estimator = FasterRCNN(
         config=config,
         writer=writer,
-        device=torch.device("cpu"),
         checkpointer=checkpointer,
         kfp_writer=kfp_writer,
+        logdir="/tmp",
```

Comment:
1. Can we have one string dir="/tmp" instead of creating multiple "/tmp"?
2. After test are you deleting this dir?

Reply:
This directory should being created by tempfile.TemporaryDirectory() i can replace the string with a constant.